frontend/src/routes/Home/Overview/OverviewPage.test.tsx

```diff
@@ -1115,8 +1115,8 @@ it('should render overview page with expected data', async () => {
 
   // Check Cluster violations chart rendered
   await waitFor(() => expect(getAllByText('Cluster violations')).toHaveLength(2))
-  await waitFor(() => expect(getByText('1 No violations')).toBeTruthy())
-  await waitFor(() => expect(getByText('1 Violations')).toBeTruthy())
+  await waitFor(() => expect(getByText('1 With no violations')).toBeTruthy())
```

Comment:
This should be lowercase.

Reply:
This page's other badges are all Capital

Reply:
Ah okay, that should be fine then.